### ISSUE TITLE
Accessibility issues in L3:02

### ISSUE THREAD

**Issue by ghost** [udacity/ud891#6]
[L3:02](http://udacity.github.io/ud891/lesson3-semantics-built-in/02-chromevox-lite/)

Departure and arrival autocomplete entries are not read aloud when navigated with the keyboard.

Furthermore they do not inherit the styling from the parent wrapper div:  `<div class="wrapper blur">`

**Comment by robdodson** [comment]
@SMEmp I see, I think you're referring to using the down/up arrow keys to select a field using ChromeVox Lite? I noticed that VoiceOver on Mac also does not read the fields aloud. This may just be a screen reader quirk. Tagging @alice in case she knows more

**Comment by alice** [comment]
Yep, this is a browser/screen reader quirk. I note this works with VO in Firefox - Safari doesn't seem to support `datalist` at all 😞 

It's not going to be feasible to fix this issue in ChromeVox Lite, unfortunately. I will definitely file a bug on Chrome regarding the VO issue and follow up here.

Since it's possible to type in the departure city, and this isn't a production form, I'm ok with leaving it as is for now, but very much appreciate the bug report!

[Edit] Here's the Chrome bug - actually filed earlier this year: http://crbug.com/618703

**Comment by robdodson** [comment]
Closing this but happen to reopen if there are additional issues.
